### ISSUE TITLE
Shapes are top-left aligned

### ISSUE THREAD

**Issue by albi005** [dotnet/maui#9089]
### Description

Shapes are always top-left aligned instead of respecting offsets. This causes [#7700](https://github.com/dotnet/maui/issues/7700).

![image](https://user-images.githubusercontent.com/22728934/181921188-9b83195a-e8c5-4a6f-a0ac-e450135c07ec.png)

Repro repo: https://github.com/albi005/MauiApp13

### Steps to Reproduce

Add this code to any page:
```xaml
<Grid BackgroundColor="Beige">
    <Path
        Data="M100 100 200 200"
        Stroke="Blue"
        StrokeThickness="5"
        Opacity=".5" />
    <Line
        X1="100"
        Y1="100"
        X2="200"
        Y2="200"
        Stroke="Red"
        StrokeThickness="20"
        Opacity=".5" />
    <Polygon
        Points="100,100 200,100 100,200"
        Fill="Green"
        Opacity=".5" />
    <Path Fill="Yellow" Opacity=".5">
        <Path.Data>
            <EllipseGeometry
                Center="150,150"
                RadiusX="50"
                RadiusY="50" />
        </Path.Data>
    </Path>
</Grid>
```
The lines and the triangle are supposed to start at 100,100 but instead they start at 0,0. The circle is supposed to be centered around 150,150.

### Version with bug

6.0.408 (current)

### Last version that worked well

Unknown/Other

### Affected platforms

iOS, Android, Windows

### Affected platform versions

all

### Did you find any workaround?

Manually setting TranslationX and TranslationY

**Comment by albi005** [comment]
The transformation that moves the Path is defined at https://github.com/dotnet/maui/blob/main/src/Controls/src/Core/Shapes/Shape.cs#L197. This applies a transformation based on the `Aspect`, so `Aspect = (Stretch)4` can be used to turn off the aligning.

**Comment by rmarinho** [comment]
fixed by #10417
